myhpom/templates/myhpom/accounts/choose_network.html

```diff
@@ -66,13 +77,13 @@ <h4>
                 <div class="row mt-4">
                     <div class="col-md-6 col-lg-8">
                         <div class="form-group">
-                            <label for="id_healthcare_network_full">Complete list of healthcare networks:</label>
-                            <select name="healthcare_network" id="id_healthcare_network_full" class="custom-select">
+                            <label for="id_health_network_full">Complete list of healthcare networks:</label>
```

Comment:
Not clear on *why* but the django docs want one to use `{% form.id_health_network_full.id_for_label %}` in this situation.